config/routes.rb

```diff
@@ -2,10 +2,14 @@
   # For details on the DSL available within this file, see http://guides.rubyonrails.org/routing.html
   mount JasmineRails::Engine => "/specs" if defined?(JasmineRails)
 
-  with_options format: false do |r|
-    r.get 'healthcheck', to: proc { [200, {}, ['']] }
-    # FIXME: Update when https://trello.com/c/w8HN3M4A/ is ready
-    r.get 'foreign-travel-advice/:country/:part' => 'travel_advice#show'
-    r.get '*path' => 'content_items#show', constraints: { path: %r[.*] }
-  end
+  get 'healthcheck', to: proc { [200, {}, ['']] }
+
+  # FIXME: Update when https://trello.com/c/w8HN3M4A/ is ready
+  get 'foreign-travel-advice/:country/:part' => 'travel_advice#show'
+
+  get '*path(.:locale)(.:format)' => 'content_items#show',
```

Comment:
It looks like the only thing that constrains this route is that it's got an /atom/ and a valid locale.

What happens when you try to render something that is not a travel advice format? Does this work?